static/app/components/events/interfaces/spans/spanDetail.tsx

```diff
@@ -257,44 +260,14 @@ class SpanDetail extends React.Component<Props, State> {
   }
 
   renderTraceButton() {
-    const {span, orgId, organization, trace, event} = this.props;
-
-    const {start, end} = getTraceDateTimeRange({
-      start: trace.traceStartTimestamp,
-      end: trace.traceEndTimestamp,
-    });
+    const {span, organization, event} = this.props;
 
     if (isGapSpan(span)) {
       return null;
     }
 
-    const orgFeatures = new Set(organization.features);
-
-    const traceEventView = EventView.fromSavedQuery({
-      id: undefined,
-      name: `Transactions with Trace ID ${span.trace_id}`,
-      fields: [
-        'transaction',
-        'project',
-        'trace.span',
-        'transaction.duration',
-        'timestamp',
-      ],
-      orderby: '-timestamp',
-      query: `event.type:transaction trace:${span.trace_id}`,
-      projects: orgFeatures.has('global-views')
-        ? [ALL_ACCESS_PROJECTS]
-        : [Number(event.projectID)],
-      version: 2,
-      start,
-      end,
-    });
-
     return (
-      <StyledDiscoverButton
-        size="xsmall"
-        to={traceEventView.getResultsViewUrlTarget(orgId)}
-      >
+      <StyledDiscoverButton size="xsmall" to={generateTraceTarget(event, organization)}>
         {t('Search by Trace')}
```

Comment:
Thoughts on rewording this to
`View Trace`?

Reply:
I do like that. I'll do that in a follow-up PR there's at least one other place that uses the same copy. Additionally, I want to keep this the same as before until the feature flags are removed, so I'll likely include the copy change in the PR that removes the flags.